ferrocene/doc/user-manual/src/rustfmt/rustfmt-cli.rst

```diff
@@ -0,0 +1,86 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Command-Line Interface
+======================
+
+.. cli:program:: rustfmt
+
+   .. cli:option:: --check
+
+      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+
+   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+
+      What data to emit and how
+
+   .. cli:option:: --backup
+
+      Backup any modified files.
```

Comment:
I'm pretty sure this flag is not covered by the hazop (or at least it wasn't last time I checked), so we should probably mention the flag isn't allowed in safety-critical contexts.

Also, this is not documenting *where* modified files are backed up.